src/Mollie/WC/Helper/PayPalButtonHandler.php

```diff
@@ -84,11 +84,17 @@ protected function whichPayPalButton()
         if(!$paypalSettings){
             return "";
         }
-        $colorSetting = $paypalSettings['color'];
-        $languageSetting = $paypalSettings['language'];
+        $colorSetting = $paypalSettings['color']?$paypalSettings['color']:"en-checkout-pill-golden";
```

Comment:
It's better to check `$paypalSettings['color']` with `isset`, like this:
```php
   isset( $paypalSettings['color']) ? $paypalSettings['color'] : "en-checkout-pill-golden"
```

otherwise, there will be notice in PHP 7 and warning in PHP 8. Unless you are absolutely sure this array element is there, of course.